l10n_it_fatturapa_in/migrations/12.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Sergio Corato <https://github.com/sergiocorato>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+from psycopg2 import sql
+
+
+def create_withholding_data_lines(env):
+    # create ftpa_withholding_ids from ftpa_withholding_type and ftpa_withholding_amount
+    column_name = openupgrade.get_legacy_name('ftpa_withholding_amount')
+    if openupgrade.column_exists(env.cr, 'account_invoice', column_name):
+        openupgrade.logged_query(
+            env.cr, sql.SQL(
+                """
+                INSERT INTO withholding_data_line
```

Comment:
Considerato che tra questa INSERT e la successiva cambia solo una colonna, potresti provare ad accorparle in una sola utilizzando le espressioni condizionali di PostgreSQL? Es. `IF` o `NULLIF`.

Reply:
Riesci a fare una PR a riguardo? Dovrei approfondire, al momento sono un po' tirato.

Reply:
https://github.com/sergiocorato/l10n-italy/pull/1

Reply:
prima di mergiare la mia PR l'hai anche testata?

Reply:
No, ho solo guardato il codice